scripts/swagger-gen.sh

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env bash
-
 SWAGGER_IMAGE="quay.io/goswagger/swagger"
-API_VERSION="v1.10.0"
-SWAGGER_FILE="https://raw.githubusercontent.com/goharbor/harbor/${API_VERSION}/api/harbor/swagger.yaml"
-
+API_VERSION_V1="v1.10.0"
```

Comment:
How about we read `API_VERSION` as `$1` from command line? So you would call `./swagger-gen v1.10.0` to generate a v1 API. We can you use this as a target in Makefile then as `make swagger-v1` and `make swagger-v2`.